### DISCUSSION TITLE
We tried an upgrade on Knowlg and faced issues

### DISCUSSION THREAD

**Discussion by darshannagesh** [Sunbird-Knowlg/Community#159] [General Discussions]
In our implementation, we did knowlg upgrade
Once this was done, all the framework APIs started failing. 
Please suggest

**Comment by pallakartheekreddy** [comment]
@darshannagesh Could you please provide additional information on the following points:
1. Which specific versions did you upgrade from and to?
2. When you mention that framework APIs began to fail, could you elaborate on the errors encountered? Have you reviewed the logs for the taxonomy service to gather more info?

**Comment by darshannagesh** [comment]
@pallakartheekreddy 
1. Version upgrade was done to release 5.6.0 version 
2. We wanted to meet a milestone and could not collect logs then. We will upgrade it again to 5.6.0 and collect the logs and share it with you.

**Reply by pallakartheekreddy** [reply]
@darshannagesh upgrade to release 5.6.0 was performed from which version?

**Reply by pallakartheekreddy** [reply]
@darshannagesh Any update on this?

**Reply by pallakartheekreddy** [reply]
@darshannagesh As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
